docs/framework/configure-apps/file-schema/winforms/index.md

```diff
@@ -20,9 +20,9 @@ Windows Forms application configuration settings are stored in an application co
 
 ```xml
 <configuration>
-  \<System.Windows.Forms.ConfigurationSection>
+  <System.Windows.Forms.ConfigurationSection>
```

Comment:
... and same thing with lines 68 and 70 in *windows-forms-add-configuration-element.md*.